gen/definitions/properties.yaml

```diff
@@ -1235,3 +1248,347 @@ topSystem:
     version: "The version for this system."
     virtualMode: "The virtual mode of this system."
+
+fvBD:
+  remove_valid_values:
+    vmac:
+      - "not-applicable"
+  add_valid_values:
+    epMoveDetectMode:
+      - ""
+  ignore_properties_in_test:
+    enableRogueExceptMac: "no"
+  default_values:
+    hostBasedRouting: "no"
+    mcastARPDrop: "yes"
+    enableRogueExceptMac: "no"
+    intersiteBumTrafficAllow: "no"
+    ipLearning: "yes"
+    limitIpLearnToSubnets: "yes"
+    mcastAllow: "no"
+    ipv6McastAllow: "no"
+  overwrites:
+    arp_flood: "arp_flooding"
+    host_based_routing: "advertise_host_routes"
+    unk_mac_ucast_act: "l2_unknown_unicast_flooding"
+    unk_mcast_act: "l3_unknown_multicast_flooding"
+    v6unk_mcast_act: "ipv6_l3_unknown_multicast_flooding"
+    ipv6_mcast_allow: "pim_ipv6"
+    multi_dst_pkt_act: "multi_destination_flooding"
+    unicast_route: "unicast_routing"
+    mac: "custom_mac_address"
+    vmac: "virtual_mac_address"
+    ll_addr: "link_local_ipv6_address"
+    mcast_allow: "pim"
+    mcast_arp_drop: "drop_arp_with_multicast_smac"
+    ep_clear: "clear_remote_mac_entries"
+    ep_move_detect_mode: "endpoint_move_detection_mode"
+    enable_rogue_except_mac: "enable_rogue_exception_mac"
+    type: "bridge_domain_type"
+  documentation:
+    OptimizeWanBandwidth: "Optimize WAN Bandwidth for the %s object."
+    arpFlood: "Enable ARP flooding for the %s object. If flooding is disabled, unicast routing will be performed on the target IP address."
+    enableRogueExceptMac: "Enable rogue exception based on MAC wildcard for the %s object."
+    epClear: "Clear all End-Points in leafs for the %s object."
+    hostBasedRouting: "Advertise host routes (/32 prefixes) out of the L3Out(s) associated to the %s object."
+    intersiteBumTrafficAllow: "Enable intersite Broadcast, Unknown-Unicast and Multicast (BUM) traffic between sites for the %s object."
+    intersiteL2Stretch: "Enable L2 stretch between sites for the %s object."
+    ipLearning: "The IP learning setting of the %s object."
+    ipv6McastAllow: "Enable IPv6 multicast traffic for the %s object."
+    limitIpLearnToSubnets: "Limit IP address learning to subnets for the %s object. Every %s object can have multiple subnets associated with it"
+    mcastARPDrop: "Drop roque multicast ARP packets for the %s object."
+    mcastAllow: "Enable IPv4 multicast traffic for the %s object."
+    bridge_domain_type: "The domain type of the %s object."
+    unicastRoute: "Enable the forwarding method based on predefined forwarding criteria (IP or MAC address)."
+    unkMcastAct: "The forwarding method for unknown layer multicast destinations."
+    v6unkMcastAct: "The forwarding method for unknown IPv6 multicast destinations."
+    vmas: "The virtual MAC address of the %s object. This is used when the the %s object is extended to multiple sites using L2 Outside."
```

Comment:
```suggestion
    vmac: "The virtual MAC address of the %s object. This is used when the the %s object is extended to multiple sites using L2 Outside."
```
typo?

Reply:
typo, changed and now the docs got updated properly also